x500/cert/base/src/test/java/org/wildfly/security/x500/cert/X509CertificateBuilderTest.java

```diff
@@ -18,7 +18,13 @@
 
 package org.wildfly.security.x500.cert;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+
```

Comment:
Looks great @Samith10! 

Just a very small nit that's about good practice in general - it's good to avoid introducing additional white space.

This is a very minor comment but it would be good to remove the additional white space that was added here.